README.md

```diff
@@ -25,6 +25,10 @@ We prepared multiple environments: [production](https://grafana.com/docs/grafana
 curl -fsSL https://raw.githubusercontent.com/grafana/oncall/dev/docker-compose.yml -o docker-compose.yml
 ```
 
+If you are attempting to run Oncall on a Raspberry Pi or other ARM64 architecture, make the following changes throughout:
+* `mysql:5.7` -> `arm64v8/mysql`
```

Comment:
Hmm, I don't see a `5.7` tag on `arm64v8/mysql`, the earliest `arm64v8/mysql` version is `8.x.x`. Grafana OnCall could potentially run on MySQL 8 (I'm not sure, haven't tested that), but I wouldn't recommend this since it's only tested on `5.7`. I could propose to create a separate issue on updating docker-compose images and helm chart to use mysql 8, as well as changing this in our CI.
Then, after updating to mysql 8 we wouldn't need to use `arm64v8/mysql`, the official mysql docker repo has arm64 images for versions `8.x.x`.

Reply:
Sounds good - thanks for your feedback! I'll cut that issue, and resolve this PR.